docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: "3.5"
 
 services:
   graphite:
-    image: graphiteapp/graphite-statsd:latest
+    image: graphiteapp/graphite-statsd:1.1.8-5
```

Comment:
I suppose that this file will go in the new ansible-scripts, correct?

For the versions, would it be OK to bind them to the minor version? Like `1.1` for graphite, and `7.5` for grafana? Or is that too dangerous?

Reply:
> I suppose that this file will go in the new ansible-scripts, correct?

Correct, that's best handle there
 
> For the versions, would it be OK to bind them to the minor version? Like `1.1` for graphite, and `7.5` for grafana? Or is that too dangerous?

Hum, the tag `1.1` doesn't exist on DockerHub, so docker-compose gives me
```
ERROR: for graphite  manifest for graphiteapp/graphite-statsd:1.1 not found: manifest unknown: manifest unknown
```